### DISCUSSION TITLE
Tasmota 13.0.1. -> no SR04 Tri and Ech

### DISCUSSION THREAD

**Discussion by grericht** [arendst/Tasmota#19595] [Q&A] [answered]
I try to connect SR04 to ESP8266. But i cant find SR04 Tri and Ech in the GPIO Configuration List for Generic(18).
i try to follow this article: https://tasmota.github.io/docs/HC-SR04/ but cant choose SR04 Tri and Ech

**Comment by sfromis** [comment, marked as answer]
You need to use a build including this feature, like upgrading to the precompiled `tasmota-sensors.bin.gz`
(That's what the box in the start of the page you linked says)
